lerna.json

```diff
@@ -1,6 +1,4 @@
 {
-  "packages": [
-    "packages/mgt"
-  ],
-  "version": "1.3.3"
+  "packages": ["packages/mgt-element", "packages/mgt"],
```

Comment:
you can change this to a wildcard so all packages in the packages folder work (`packages/**/*`)

Reply:
Good idea! I set to be explicit at one point to try and influence the order in which lerna runs commands against the packages. Turns out to be unnecessary though.